core/src/test/java/org/wildfly/extras/sunstone/api/impl/docker/DockerNodeTest.java

```diff
@@ -535,6 +538,23 @@ public void testInsufficientCapabilities() {
         }
     }
 
+    @Test
+    public void testVolumeBindings() {
+        final int nonce = new Random().nextInt();
+        try (DockerNode node = (DockerNode) dockerProvider.createNode("alpineVolumes",
+                ImmutableMap.<String, String> of("template", "alpine", "docker.binds", "/tmp:/data"))) {
+            final ExecResult execResult = node.exec("sh", "-c", "echo " + nonce + " > /data/testVolumeBindings");
```

Comment:
Ah, I couldn't understand why you were using `node.exec()` everywhere. Now I do. OK, no problem with that.